OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -326,6 +352,9 @@ public override Activity Tick(Actor self)
 					moveFraction = MoveFractionTotal;
 				UpdateCenterLocation(self, Move.mobile);
 
+				if (IsCanceled)
+					ret.NextActivity = Move.NextActivity;
+
```

Comment:
I am pretty sure canceling a MovePart is a bad idea, @pchote correct me if I am wrong.